improver/cli/__init__.py

```diff
@@ -203,16 +203,16 @@ def inputpath(to_convert):
 def create_constrained_inputcubelist_converter(*constraints):
     """Makes function that the input constraints are used in a loop.
 
-    The function is value_converter, this means it is used by clize to convert
+    The function is a @value_converter, this means it is used by clize to convert
     strings into objects.
     This is a way of not using the IMPROVER load_cube which will try to merge
     cubes. Iris load on the other hand won't deal with meta data properly.
-    So an example if you wanted to load an X cube and a Y cube from a cubelist
+    So an example is if you wanted to load an X cube and a Y cube from a cubelist
     of 2. You call this function with a list of constraints.
     These cubes get loaded and returned as a CubeList.
 
     Args:
-        *constraints (tuple of str or list):
+        *constraints (callable):
             Constraints to be used in the loading of cubes into a CubeList.
             If the tuple contains a string or multiple strings, then each
             string is expected to return exactly one match. If the tuple
```

Comment:
This needs updating too. Something like "Constraints to be used in extracting the required cubes. Each constraint must match exactly one cube and extracted cubes will be sorted to match their order. A constraint can be an `iris.Constraint` object or a callable or cube name that can be used to construct one.